.github/workflows/conflicts.yml

```diff
@@ -42,9 +42,7 @@ jobs:
       - name: Install PyMedPhys with propagate environment
         if: steps.conflicts.outputs.status == 'true'
         run: |
-          poetry install -E propagate
-          pip install pipx
-          pipx install poetry-merge-lock
+          poetry install -E propagate || poetry lock --no-updates && poetry install -E propagate
```

Comment:
What is the order/grouping of operations.?
I tend to put parentheses to make it obvious

Reply:
That would make the commands run in a subshell. (https://unix.stackexchange.com/a/26069). Does that still work here?

I didn't think bash had order of operations... It was my impression that "&&" isn't really "and" and "||" isn't really "or", but instead `||` means, run the following if the previous failed, and `&&` means, run the following if the previous passed, and `;` means run the following regardless...